### PR TITLE
Fix/Core#573 - Wrong doc for the type of `read_fct_params` of GenericDataNode on Taipy 2.2

### PR DESCRIPTION
**Pull request by trgiangdo**
https://github.com/Avaiga/taipy-core/issues/573

### CODE REVIEW

File: docs/manuals/core/config/data-node-config.md [hunk 1]
```diff
@@ -739,10 +739,10 @@ can be provided:
   More optional parameters can be passed through the _**write_fct_params**_ parameter.
 
 - _**read_fct_params**_ represents the parameters passed to the _read_fct_ to
-  read/de-serialize the data. It must be a `List` type object.
+  read/de-serialize the data. It must be a `Tuple` type object.
```
Comment (toan-quach) [inline docs/manuals/core/config/data-node-config.md:742]:
```suggestion
  read/de-serialize the data. It must be a `List` type object.
```
This has been changed back to List for simplicity on the user's understanding :D

File: docs/manuals/core/config/data-node-config.md [hunk 2]
```diff
@@ -739,10 +739,10 @@ can be provided:
...
 
 - _**write_fct_params**_ represents the parameters passed to the _write_fct_ to write
-  the data. It must be a `List` type object.
+  the data. It must be a `Tuple` type object.
```
Comment (toan-quach) [inline docs/manuals/core/config/data-node-config.md:745]:
```suggestion
  the data. It must be a `List` type object.
```
This has been changed back to List

**Review by toan-quach — CHANGES_REQUESTED** [review]
lgtm, just minor changes required!

**Review by toan-quach — CHANGES_REQUESTED** [review]
lgtm, just minor changes required!

File: docs/manuals/core/config/data-node-config.md (hunk 1 above)
Comment (trgiangdo) [reply docs/manuals/core/config/data-node-config.md:742]: On taipy 2.2, we only allow Tuple. List will be for 2.3

**Review by toan-quach — APPROVED** [review]

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**